### PR TITLE
Document  key for exercise  files

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: building/tracks/concept-exercises.md [hunk 1]
```diff
@@ -312,6 +312,8 @@ This file contains meta information on the exercise:
 - `blurb`: A short description of this exercise. Its length must be <= 350. Markdown is _not_ supported (required)
 - `source`: The source this exercise is based on (optional)
 - `source_url`: The URL of the source this exercise is based on (optional)
+- `representer`: Meta information related to how the representer processes this file (optional)
+  - `version`: An integer for the version of the representer to use for the exercise (required if parent key is present)
```
Comment (ErikSchierboom) [inline building/tracks/concept-exercises.md:316]: Maybe:
> An integer which can be incremented to invalidate representations (required if parent key is present - defaults to 1 downstream if missing, so set to 2 to initially invalidate).

**Review by ErikSchierboom — COMMENTED** [review]
CC @ee7

**Review by iHiD — APPROVED** [review]

File: building/tracks/concept-exercises.md [hunk 2]
```diff
@@ -348,7 +350,10 @@ Assume that the user `FSharpForever` has written an exercise called `log-levels`
   "language_versions": ">=3.7",
   "blurb": "Learn how to work with strings by processing log lines.",
   "source": "Wikipedia",
-  "source_url": "https://en.wikipedia.org/wiki/Log_file",
+  "source_url": "https://en.wikipedia.org/wiki/Log_file",,
```
Comment (ee7) [inline building/tracks/concept-exercises.md]:
```suggestion
  "source_url": "https://en.wikipedia.org/wiki/Log_file",
```

**Review by ee7 — APPROVED** [review]

File: building/tracks/concept-exercises.md (hunk 1 above)
Comment (ee7) [reply building/tracks/concept-exercises.md:316]: (Ah, here was the famous reply-via-overwriting-edit).

Parsing an omitted `version` property the same as an explicit `"version": 1` sounds OK to me.

Note that `configlet lint` leaned towards not erroring for an empty optional key, though. So maybe `"representer": {}` shouldn't be an error. But perhaps removed with `configlet fmt`.

**Pull request merged**